### PR TITLE
feat(docs): add release manager runbook

### PR DESCRIPTION
**Pull request by maggieneterval**
* feat(docs): add release manager runbook 

* feat(docs): add release manager to release page

### CODE REVIEW

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md]: Currently, I think only the Google team has access to log into GitHub as spinnaker-release. I'm happy to do these steps for Ethan this time around, but should we consider giving access to all Spinnaker release managers?

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md]: Similarly, this step jumped out to me as one that only Google team members can do (I think?). Not sure this is a huge deal, just wanted to highlight.

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [inline community/releases/release-manager-runbook/index.md]: @ttomsu This link is no longer valid, I'm guessing because it got moved into the sponnet project, but just wanted to double-check with you that this is the same `publish.sh`: https://github.com/spinnaker/sponnet/blob/master/publish.sh

File: community/releases/release-manager-runbook/index.md
Comment (ezimanyi) [inline community/releases/release-manager-runbook/index.md]: Should this be the oldest flow instead of the second-oldest one?

File: community/releases/release-manager-runbook/index.md
Comment (ezimanyi) [inline community/releases/release-manager-runbook/index.md]: I think this one runs several times per day.

File: community/releases/release-manager-runbook/index.md
Comment (ezimanyi) [inline community/releases/release-manager-runbook/index.md]: I usually use something like [this](https://github.com/pulls?utf8=%E2%9C%93&q=org%3Aspinnaker+is%3Apr+is%3Aopen+-base%3Amaster) because it works even if the `cherry-pick` label isn't applied.

File: community/releases/release-manager-runbook/index.md
Comment (ezimanyi) [reply community/releases/release-manager-runbook/index.md]: Yes, I think it makes sense for everyone to have the same access (even this time).

File: community/releases/release-manager-runbook/index.md
Comment (ezimanyi) [reply community/releases/release-manager-runbook/index.md]: I think it should be easy to add people outside of Google as admins of this group, as the group is not in the Google domain.

File: community/releases/release-manager-runbook/index.md
Comment (ezimanyi) [inline community/releases/release-manager-runbook/index.md]: I think this line break breaks the formatting.

File: community/releases/release-manager-runbook/index.md
Comment (ezimanyi) [inline community/releases/release-manager-runbook/index.md]: I think this needs a newline above to format correctly

**Review by ezimanyi — APPROVED** [review]
This looks great!  Thanks so much for moving these to a better location! 💯 🥇 📜

File: community/releases/release-manager-runbook/index.md
Comment (ttomsu) [reply community/releases/release-manager-runbook/index.md]: You may want to consider transferring the doc with the 2FA image to the `spinnaker.io` domain, maybe owned by admin@spinnaker.io

File: community/releases/release-manager-runbook/index.md
Comment (ttomsu) [reply community/releases/release-manager-runbook/index.md]: Sure looks right to me.

File: community/releases/release-cadence/index.md
Comment (ttomsu) [inline community/releases/release-cadence/index.md]: `Release Manager (Slack ID)`

File: community/releases/release-manager-runbook/index.md
Comment (plumpy) [inline community/releases/release-manager-runbook/index.md]: I think this URL will work better: https://github.com/pulls?q=is%3Apr+author%3Aspinnakerbot+is%3Aopen

And the text should probably be updated since there are many more besides kork and they should automerge, so any open ones by definition had a problem that needs to be fixed. So maybe "If there are any outstanding autobump PRs, make the required fixes to allow them to merge." Or something?

Helpfully, in creating that search, I found a rosco version bump that was just lying around and fixed it :)

File: community/releases/release-manager-runbook/index.md
Comment (plumpy) [inline community/releases/release-manager-runbook/index.md]: The last release was not converted to camel case and apparently nothing blew up, so should we remove that suggestion? It's "Gilmore Girls A Year in the Life"

File: community/releases/release-manager-runbook/index.md
Comment (plumpy) [inline community/releases/release-manager-runbook/index.md]: @ttomsu Doesn't Spinnakerbot get auto-deployed now that it's a cloud run job?

**Review by plumpy — APPROVED** [review]
Awesome!!

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: Link to Slack dev channel?

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: Do you have a role that you could use here, by name? Like instead of "I'll be cutting...on Tuesday" it would be "Build czar cuts...on Tuesday"

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: Same here. s/I'll/<some role>/

File: community/releases/release-manager-runbook/index.md
Comment (ezimanyi) [reply community/releases/release-manager-runbook/index.md]: Well, if only you knew the pain I caused by initially calling it "Gilmore Girls: A Year in the Life"...turns out we interpolate it directly into YAML...

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: I think these sub-step numberings all want to be `1. `
Jekyll renders them as a., b., c., etc.

Also, the supersteps, you can make those all `1.` also, which makes it easier if someone edits to add a step.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: What is "soon" here?

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
7. At this point, the following Flow_BuildAndValidate_* jobs should exist:
```

File: community/releases/release-manager-runbook/index.md
Comment (plumpy) [reply community/releases/release-manager-runbook/index.md]: Well, I have just the PR to fix that, by moving the publish code into spinrel. Hopefully ready before the 1.20.0 release.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: Change these letters to bullets.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
10. Share a curated changelog with any partners from the community who want to
```

File: community/releases/release-manager-runbook/index.md
Comment (ttomsu) [inline community/releases/release-manager-runbook/index.md]: against deprecated changelog file?

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Sure, I will also update the example to "Gilmore Girls A Year in the Life" so that release name is immortalized.

File: community/releases/release-manager-runbook/index.md
Comment (ttomsu) [reply community/releases/release-manager-runbook/index.md]: Yes - spinbot runs right off the `master` branch now, so this section should be deleted

**Review by ttomsu — APPROVED** [review]

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Oh that's right! I'll delete this section.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    major-minor part of this version number should match the Gate version for
```

Comment (dorbin) [reply community/releases/release-manager-runbook/index.md]: Or "major/minor"

Or "major.minor"

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: I think just one backtick, for inline. I'm not sure how this would render with the three backticks.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [reply community/releases/release-manager-runbook/index.md]: ^^ for all steps/sub-steps in the doc

**Review by dorbin — COMMENTED** [review]
Great doc, Maggie!

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: sure!

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Ah good catch. Going to move the parenthetical statements out of the headers for each section and into regular text just below.

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: 👍 We usually call the role "the release manager."

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Great to know, thanks! Updating.

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Updated "soon" to "in the following step"

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Updating.

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Well, the spinnaker-release password lives in the internal password tool, so I guess we'd need to share that too...going to defer to @plumpy on whether sharing that password is okay?

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: You are correct! Just added Ethan as a `manager`. I'll update these instructions that new release managers will need to be added to this group.

**Comment by maggieneterval** [comment]
Thanks all for the thorough review and helpful comments 🤗

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: Now that I look again...
There's only this one step under this heading, so you should probaby just make it a regular paragraph.

File: community/releases/release-manager-runbook/index.md
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: Here too

**Review by dorbin — APPROVED** [review]

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Updated all the single-step sections to just use paragraphs!

File: community/releases/release-manager-runbook/index.md
Comment (maggieneterval) [reply community/releases/release-manager-runbook/index.md]: Let's continue the conversation on how best to handle spinnaker-release permissions in #build-process; going to merge this PR as-is for now but of course happy to update if needed when we land on a solution here.

**Pull request merged**
